tests/integration/feature_store_integration/test_feature_materialize.py

```diff
@@ -77,8 +77,34 @@ def features_fixture(event_table):
         feature_names=["EXTERNAL_FS_COUNT_OVERALL_7d"],
     )["EXTERNAL_FS_COUNT_OVERALL_7d"]
 
+    # Dict feature
+    cross_aggregate_feature1 = event_view.groupby(
+        "ÜSER ID", category="PRODUCT_ACTION"
+    ).aggregate_over(
+        "ÀMOUNT",
+        method="sum",
+        windows=["7d"],
+        feature_names=["amount_sum_across_action_7d"],
+    )[
+        "amount_sum_across_action_7d"
+    ]
+
+    cross_aggregate_feature2 = event_view.groupby(
+        ["CUST_ID"], category="PRODUCT_ACTION"
+    ).aggregate_over(
+        "ÀMOUNT",
+        method="sum",
+        windows=["24d"],
+        feature_names=["amount_sum_across_action_24d"],
+    )[
+        "amount_sum_across_action_24d"
+    ]
+
+    feature_6 = cross_aggregate_feature1.cd.cosine_similarity(cross_aggregate_feature2)
+    feature_6.name = "EXTERNAL_FS_COSINE_SIMILARITY"
+
     # Save all features to be deployed
-    features = [feature_1, feature_2, feature_3, feature_4, feature_5]
+    features = [feature_1, feature_2, feature_3, feature_4, feature_5, feature_6]
```

Comment:
Would be good to also check for a count dict feature that is not post-processed into numeric type.

Reply:
If i understood the request here correctly, do you mean check for a count dict feature that is post-processed into non-numeric type? For post-processed into non-numeric type, I need lookup feature & currently it is not supported (directly calling the method like `key_with_highest_value` or `most_frequent` do not trigger ODFV construction, method like `get_value` or `get_rank` require lookup feature.).

Reply:
Oh actually I meant simply using the dictionaries as features without post-processing.